specification/serialconsole/resource-manager/Microsoft.SerialConsole/stable/2018-05-01/examples/DisableConsoleExamples.json

```diff
@@ -1,19 +1,20 @@
 {
   "parameters": {
     "api-version": "2018-05-01",
-    "subscriptionId": "00000000-00000-0000-0000-000000000000",
-    "default": "default"
```

Comment:
I believe "default" is actually modeled as a parameter.  Typically this would be the session or instance, but since there is only 1 subscription and it is self-describing this is effectively always "hardcoded" for this API